pyproject.toml

```diff
@@ -1,10 +1,23 @@
-#[project]
-#requires-python = ">=3.9"
-
 [build-system]
-requires = ["setuptools>=62.4.0"]
+requires = [
+    "Babel>=2.10.0",
+    "PyInstaller==6.12.0",
+    "pytest>=8.1",
+    "setuptools>=62.4.0",
+]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "musicbrainz-picard"
+description = "MusicBrainz Picard"
+authors = [{name = "The MusicBrainz Team"}]
+license = {text = "GPL-2.0-or-later"}
+keywords = ["musicbrainz", "tagging", "audio"]
+dynamic = ["version", "readme", "classifiers", "requires-python", "dependencies"]
```

Comment:
I think for this only "version", "readme" and for now "dependencies" should be listed.

"readme" can then be set in `tool.setuptools.dynamic` as 

```toml
readme = {file = ["README.md"], content-type = "text/markdown"}
```

and the corresponding code be removed from `setup.py`.

"requires-python" and "classifiers" should be set above as static data (and removed from setup.py.

Reply:
I just found out that we can also set `dependencies` in `tool.setuptools.dynamic` as 

```
dependencies = {file = ["requirements.txt"]}
```

This allows us to also remove `install_requires` from `setup.py`.

Reply:
But I guess we will rework dependencies when we move to e.g. ruff, so we can keep it in `setup.py` or move it here for now. Doesn't matter much.